client/my-sites/stats/activity-log-day/placeholder.js

```diff
@@ -0,0 +1,26 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import FoldableCard from 'components/foldable-card';
+
+const component = (
+	<FoldableCard
+		className="activity-log-day__placeholder"
+		header={
+			<div>
+				<div className="activity-log-day__day" />
+				<div className="activity-log-day__events" />
+			</div>
+		}
+	/>
+);
+
+export default function ActivityLogDayPlaceholder() {
```

Comment:
Personal preference 🙂 

Simple top-level functions that don't need any special binding seem just fine as a classic `function` declaration. The hoisting can be nice sometimes but isn't relevant here.

I'd probably steer away from `export default () => {}` as I don't think it's going to get a decent name if it needs to appear in a stack trace. I don't believe the other fat arrow has that problem.